programming/autonomous.md

```diff
@@ -0,0 +1,43 @@
+#Creating an Autonomous
+
+Autonomous is code that is used to make the robot move by itself. 
+Our autonomous mainly uses a Boilerplate to create states for the robot to follow.
+We utilize the AutonomousStateMachine from magicbot to create a progression of commands for the robot to follow.
+
```

Comment:
- Can you mention where the auto file must be created?
- When you are mentioning code things, you usually want to use backticks, for example `AutonomousStateMachine`